### PR TITLE
ENH prepare inputs for Y6 sims

### PR DESCRIPTION
**Pull request by beckermr**
This PR grabs some Y3 changes we need to make the Y6 sims.

### CODE REVIEW

File: desmeds/desdm_maker.py [hunk 1]
```diff
@@ -593,7 +593,7 @@ class to prepare inputs for the DESDM version
         - write psf map file
         - write file config
     """
-    def __init__(self, medsconf, tilename, band):
+    def __init__(self, medsconf, tilename, band, skip_nullwt=False):
```
Comment (beckermr) [inline desmeds/desdm_maker.py:596]:
```suggestion
    def __init__(self, medsconf, tilename, band):
```

File: desmeds/coaddinfo.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def download(self):
             self['userstring'] = ''
 
         cmd=_DOWNLOAD_CMD % self
-
+        print("rsync command: %s", cmd)
```
Comment (beckermr) [inline desmeds/coaddinfo.py:82]:
```suggestion
```

File: bin/desmeds-prep-tile [hunk 3]
```diff
@@ -60,6 +66,7 @@ if __name__ == '__main__':
         medsconf,
         args.tilename,
         args.band,
+        skip_nullwt=args.skip_nullwt,
```
Comment (beckermr) [inline bin/desmeds-prep-tile:69]:
```suggestion
```

File: bin/desmeds-prep-tile [hunk 4]
```diff
@@ -41,6 +41,12 @@ parser.add_argument(
     help=('remove all the generated nullwt files'),
 )
 
+parser.add_argument(
+    '--skip_nullwt',
+    action='store_true',
+    help=('do not generate nullwt files'),
+)
+
```
Comment (beckermr) [inline bin/desmeds-prep-tile:49]:
```suggestion
```

File: desmeds/files.py [hunk 5]
```diff
@@ -220,6 +220,7 @@ def get_meds_base():
     # return '$MEDS_DIR'
 
     dir = os.environ['MEDS_DIR']
+    print("dir in get_meds_base:",dir)
```
Comment (beckermr) [inline desmeds/files.py:223]:
```suggestion
```

**Comment by beckermr** [comment]
@esheldon When you are back, this is ready for review!

**Comment by esheldon** [comment]
what happened to this one?

**Comment by beckermr** [comment]
After looking very hard at the codes, I realized we need something else.

**Pull request closed without merging**